src/client/chowly-api-client.ts

```diff
@@ -0,0 +1,32 @@
+import { AxiosInstance, AxiosResponse } from 'axios';
+import { plainToClass } from 'class-transformer';
+import { IConfiguration, ISections } from '../interface';
+import { Sections } from '../model';
+import { createAxiosInstance } from '../utils';
+
+export class ChowlyApiClient {
+    readonly client: AxiosInstance;
+    private readonly config: IConfiguration;
+    private basePrefix: string = '/api/toos/v2';
```

Comment:
If I'm not mistaken Axios has special option for this `prefix: '/api/toos/v2'` I guess

Reply:
axios interface doesn't have prefix, i tried some varianties, but it seems like it doesn't work, the only thing that worked for me was url, but it overrides /menu path